addon/mirage/utils.js

```diff
@@ -66,8 +75,17 @@ let pipelines = {
 
   video(file, metadata) {
     let video = document.createElement('video');
-    return new RSVP.Promise(function (resolve) {
+    return new RSVP.Promise(function (resolve, reject) {
       video.addEventListener('loadeddata', resolve);
+      video.onerror = () => {
+        reject(
+          new Error(
+            'You tried to upload an invalid video. The upload handler for mirage ' +
+            'shipped with ember-file-upload does not support invalid videos. ' +
+            'Please make sure that your video is valid and could be parsed by browser.'
```

Comment:
Tiny grammar fix:
"and could be parsed by browser" should be changed to "and can be parsed by browsers."